packages/flutter_tools/lib/src/flutter_project_metadata.dart

```diff
@@ -15,6 +15,8 @@ enum FlutterProjectType {
   /// It is different than the "module" template in that it exposes and doesn't
   /// manage the platform code.
   app,
+  /// A List/Detail app template that follows community best practices.
+  list_detail_app,
```

Comment:
nit: Dart style would be `listDetailApp`

Reply:
Sure, I can understand that is a bit different from the usual Dart Style. However, this enum appears to be the "type safe" name for the template.

Right now, to create the new template, you run the following command:

```
flutter create -t list_detail_app my_app
```

If we change this up based on your suggestion, the CLI would change to:

```
flutter create -t listDetailApp my_app
```

I don't have a strong preference either way, but wanted to highlight the CLI change and verify that camelCase is the correct case to use there.

Reply:
Maybe @jonahwilliams can clarify, but I don't understand why the string provided on the command line has to exactly match the identifier in Dart code. Something a bit too clever is probably happening somewhere if that is the case.

Reply:
With the rename from `list_detail_app` to `skeleton` this is no longer a cause of concern! This now reads `skeleton` instead, conforming to the standard conventions.

Thanks for pointing it out :)